### PR TITLE
registration error in console fixed

### PR DESCRIPTION
**Pull request by amanuela97**

### CODE REVIEW

File: pages/register.vue [hunk 1]
```diff
@@ -99,7 +99,11 @@ export default {
       this.$router.push(`/${this.profile.username}`)
     },
     async submit({ email, username, password, place }) {
-      if (!email.trim() || !username.trim() || !password.trim()) {
+      if (
+        !(typeof email !== 'string' ? '' : email.trim()) ||
```
Comment (razbakov) [inline pages/register.vue:103]: Maybe something like:


```
 !email || !email.trim() || ...
```

It's a bit shorter and could also work, what do you think?

**Review by razbakov — CHANGES_REQUESTED** [review]

File: pages/register.vue (hunk 1 above)
Comment (amanuela97) [reply pages/register.vue:103]: yes, it is a lot more efficient

File: pages/register.vue [hunk 2]
```diff
@@ -99,7 +99,14 @@ export default {
       this.$router.push(`/${this.profile.username}`)
     },
     async submit({ email, username, password, place }) {
-      if (!email.trim() || !username.trim() || !password.trim()) {
+      if (
+        !email ||
+        email.trim() ||
```
Comment (razbakov) [inline pages/register.vue:104]: should be `!email.trim()`

**Review by razbakov — CHANGES_REQUESTED** [review]

**Review by razbakov — APPROVED** [review]
LGTM

**Pull request merged**